server/sonar-server/src/main/java/org/sonar/server/computation/ws/CeWsModule.java

```diff
@@ -25,16 +25,16 @@
   @Override
   protected void configureModule() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Protected method 'configureModule(...)' is never used. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AUnusedProtectedMethod)